joi-validations/topics.validations.js

```diff
@@ -1,4 +1,4 @@
-const {string, url, array} = require('./general.validations');
+const {string, url, array, boolean} = require('./general.validations');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `boolean` is a new addition and it seems to be correctly imported from `general.validations`. This change is necessary for the new `withMinimumFollower` field in the `getTopic` validation schema.
```diff
- const {string, url, array} = require('./general.validations');
+ const {string, url, array, boolean} = require('./general.validations');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->